xt_training/utils/training.py

```diff
@@ -22,7 +23,7 @@ def default_exit(config, runner, save_dir):
 
 def train(args):
     config_path = args.config_path
-    save_dir = args.save_dir
+    save_dir = os.getenv("NNI_OUTPUT_DIR", args.save_dir)
```

Comment:
So is it easy to run NNI but save everything to a ./checkpoints directory in the repo?